### ISSUE TITLE
[Help] GET request not going through

### ISSUE THREAD

**Issue by agharium** [reactiveui/refit#724]
Hi all!

I have in my hands somekind of a pesky situation.

This is my code:

```csharp
public interface IIBGEAPI
{
    [Get("/localidades/estados")]
    Task<List<IBGEState>> GetStates();

    [Get("/localidades/estados/{id}/municipios")]
    Task<List<IBGECity>> GetStateCities([AliasAs("id")]int id);
}
```

```csharp
Task.Run(async () => {
    IIBGEAPI IBGE_API = RestService.For<IIBGEAPI>("https://servicodados.ibge.gov.br/api/v1");

    Debug.WriteLine("BEGINS"); // This appears on my log
    Debug.WriteLine(await IBGE_API.GetStates());
    Debug.WriteLine("ENDS"); // This never appears on my log
});
```

Why it's not working? It just keeps on somekind of a halt and the request doesnt ever comes back.

The same URL works perfectly on my browser:

![image](https://user-images.githubusercontent.com/28984633/62533830-0261bc00-b81e-11e9-8b3c-5fd97eb644b3.png)

I've used the `GetStates` method for this example of my issue, but the `GetStateCities` method doesn't work either. Also, I have tried putting my code on a `Task` returning method (instead of a `Task.Run` block), but it didn't work either way.

So, anyone has any idea of what could be going on? How can I debug this?

Thanks all for the patience and pre-disposition in helping :)

**Comment by rdavisau** [comment]
It might be that your call is throwing an exception that is being 'ignored' due to the outer `Task.Run` block. 

You could try wrapping the call to `await IBGE_API.GetStates()` in a try/catch and breaking in the catch handler or logging the exception to see if that's the case. If so the exception should give you a better idea of what the issue might be.

**Comment by agharium** [comment]
@rdavisau Huge thanks for the input. Also, for debugging purposes I changed the method's return type to `HttpResponseMessage` (as https://reactiveui.github.io/refit/ suggests) and I was able to get some extra info on what was going on.

It ended up being something weird about the server API that I was trying to query. I changed to another API server (what I was looking for was public information) and voilà.. working perfectly.
